### PR TITLE
[novelanheatpump] Add port config parameter (for #3607)

### PR DESCRIPTION
**Pull request by watou**
Added port config parameter for #3607.

### CODE REVIEW

**Comment by watou** [comment]
@hakan42, would you mind giving this PR a read through before it's merged?  OK to merge if you think it's ready.  Thank you.

File: bundles/binding/org.openhab.binding.novelanheatpump/src/main/java/org/openhab/binding/novelanheatpump/internal/HeatPumpBinding.java [hunk 1]
```diff
@@ -85,18 +90,32 @@ public void activate() {
...
-				// only do something if the ip has changed
-				HeatPumpBinding.ip = ip;
+			boolean properlyConfigured = true;
+
+			long refreshInterval = DEFAULT_REFRESH_INTERVAL;
+			int port = DEFAULT_PORT;
+			String ip = null;
+			try {
+				String refreshIntervalString = (String) config.get("refresh"); //$NON-NLS-1$
+				refreshInterval = StringUtils.isNotBlank(refreshIntervalString) ?
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.novelanheatpump/src/main/java/org/openhab/binding/novelanheatpump/internal/HeatPumpBinding.java]: for the sake of readability you could statically import isNotBlank()

File: distribution/openhabhome/configurations/openhab_default.cfg [hunk 2]
```diff
@@ -817,11 +817,14 @@ ntp:hostname=ptbtime1.ptb.de
...
-# IP address of the Novelan (Siemens) Heatpump to connect to
+# IP address of the Novelan (Siemens) Heatpump to connect to (required)
 #novelanheatpump:ip=
 
+# port number of the Novelan (Siemens) Heatpump to connect to (optional, defaults to 8888)
+#novelanheatpump:port=8889
+
 # refresh interval in milliseconds (optional, defaults to 60000)
-#novelanheatpump:refresh=
+#novelanheatpump:refresh=60000
```
Comment (teichsta) [inline distribution/openhabhome/configurations/openhab_default.cfg]: i would remove the value since 60000 is the default value anyhow

**Comment by teichsta** [comment]
Thanks for this contribution @watou. See some minor comments inline …

**Comment by watou** [comment]
Thank you, @teichsta.  Updated and squashed.

**Comment by teichsta** [comment]
great thanks @watou!

**Pull request merged**